changes.tex

```diff
@@ -1,3 +1,8 @@
+\subsection{Version \version}
+\begin{itemize}
+\item Remove experimental and barely used Logic attribute annotations.
+Use standard C attributes instead.
```

Comment:
By "standard C attributes", I suppose this means C23's `[[attributes]]` ? Would it be useful to mention that we are talking about them and not about GCC-style compiler attributes?

Reply:
Well, I was meaning both in fact (depending on your tooling of course). I'll clarify that.

Reply:
Done in [37c17fd](https://github.com/acsl-language/acsl/commit/37c17fdc3e927b5588ce4069ecb0a1238846d3b2)